backends/cxxrtl/cxxrtl.cc

```diff
@@ -1947,6 +1947,8 @@ struct CxxrtlWorker {
 						// attribute regardless of input.
 						case RTLIL::STi:
 							log_assert(false);
+							// unreachable, but compiler still warns about implicit fall-through
```

Comment:
This seems like a bug we should fix one way or another. Either figure out why the warning persists, or add `log_unreachable()`.

Reply:
With `NDEBUG=1` this warning is completely legit, since `log_assert` becomes a no-op. Maybe using `log_abort` would be a better choice here?

Reply:
Oh, I guess you could order this case the last to avoid the fallthrough.

Reply:
That works too, done.